pom.xml

```diff
@@ -49,6 +57,18 @@
             <version>20180130</version>
         </dependency>
 
+        <dependency>
```

Comment:
Do not think we have or should have logging in this repository. If something fails we should throw exceptions with messages to the main application may log them.

Reply:
Some dependency I added started giving out warning of logging binders so I added it to make it stop. Maybe the warnings were there before, I'm not sure.

Reply:
Took a look at this again and it was only  the executor tests giving this warning. I decided to ignore this warning and remove the logger.